src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,16 @@
 public class CliSyntax {
 
     /* Prefix definitions */
```

Comment:
Good that you indicated that the prefixes are the same. Perhaps it would be helpful to write down one more line of comment to show why it would be ok for them to be the same
```suggestion
    /* Prefix definitions */
    // One prefix can be used for multiple reasons because they are imported by their respective commands.
```

Reply:
Ok added